### PR TITLE
Docs about deleting tags

### PR DESCRIPTION
**Pull request by 0xmichalis**
@soltysh PTAL

Depends on https://github.com/openshift/origin/pull/4120 and https://github.com/openshift/origin/pull/5319

### CODE REVIEW

File: architecture/core_concepts/builds_and_image_streams.adoc [hunk 1]
```diff
@@ -401,3 +401,24 @@ for the tag "2.0":
...
 
 `oc tag ruby:latest ruby:2.0`
+
+[[tag-removal]]
+
+=== Tag Removal
+You can stop tracking a tag simply by removing it. For example, you can stop
+tracking the *latest* tag you set above:
+
+`oc tag ruby:latest -d`
```
Comment (soltysh) [inline architecture/core_concepts/builds_and_image_streams.adoc]: I'd move the -d after tag, just for purity :wink

File: cli_reference/cli_by_example_content.adoc [hunk 2]
```diff
@@ -251,14 +251,17 @@
 
 [options="nowrap"]
 ----
-  // Tag the current image for the image stream 'openshift/ruby' and tag '2.0' into the image stream 'yourproject/ruby with tag 'tip':
+  // Tag the current image for the image stream 'openshift/ruby' and tag '2.0' into the image stream 'yourproject/ruby with tag 'tip'.
```
Comment (soltysh) [inline cli_reference/cli_by_example_content.adoc]: Please leave the colons the way there were. That's the right way to do it, explain and then show the actual command (applies to all changes in here).

**Comment by soltysh** [comment]
@kargakis nits, but generally LGTM.
@adellape once the aforementioned PRs got it (just tagged them for merge) you can pick this up.

File: cli_reference/cli_by_example_content.adoc (hunk 2 above)
Comment (kargakis) [inline cli_reference/cli_by_example_content.adoc]: All the comments in the examples we have in Origin end in full stop and these examples were the only using colons.

File: architecture/core_concepts/builds_and_image_streams.adoc (hunk 1 above)
Comment (kargakis) [inline architecture/core_concepts/builds_and_image_streams.adoc]: You mean before

Comment (soltysh) [inline architecture/core_concepts/builds_and_image_streams.adoc]: I mean like that: `oc tag -d ruby:latest`.

File: cli_reference/cli_by_example_content.adoc (hunk 2 above)
Comment (soltysh) [inline cli_reference/cli_by_example_content.adoc]: If that's so, then I'm ok.

File: architecture/core_concepts/builds_and_image_streams.adoc (hunk 1 above)
Comment (kargakis) [inline architecture/core_concepts/builds_and_image_streams.adoc]: That's what I understood from the start. Updated

**Comment by kargakis** [comment]
https://github.com/openshift/origin/pull/5319

**Comment by kargakis** [comment]
@adellape ping

**Comment by adellape** [comment]
LGTM

**Pull request merged**